test/simperium/channel_test.js

```diff
@@ -420,7 +420,7 @@ describe( 'Channel', function() {
 			beforeEach( ( done ) => {
 				var data = { title: 'hola mundo' };
 
-				channel.on( 'acknowledge', function() {
+				channel.once( 'acknowledge', function() {
```

Comment:
Without this, any future `channel.emit('acknowledge')` causes a test to error because `done` is called more than once.